SU-SWS/decanter
# add favicon to style guide

# READY FOR REVIEW

# Summary
- Adds Favicon to our style guide site (decanter.stanford.edu), which currently does not have one

# Needed By (Date)
- N/A

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Compile styleguide: `npm run styleguide`
3. Verify that your local site now has a Favicon (Stanford's)

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- N/A

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
